integration-test/env/Extension/bin/_placeholder

```diff
@@ -0,0 +1 @@
+placeholder for git to preserve this directory
```

Comment:
Seems unnecessary? the COPY command in the docker file creates this directory if it doesn't exist...